docs/advanced/prometheus.md

```diff
@@ -0,0 +1,227 @@
+   
+   ```
+
+   3. install edgemesh
+
+      ```
+      git clone https://github.com/kubeedge/edgemesh.git
+      cd edgemesh
+      
+      kubectl apply -f build/crds/istio/
+      
+      Configure PSK and Relay Node 
+      vim 04-configmap.yaml
+      
+        relayNodes:
+        - nodeName: masternode ## your relay node name
+          advertiseAddress:
+          - x.x.x.x ## your relay node ip
+      
+      kubectl apply -f build/agent/resources/
+      ```
+      
+      ![image-20240329154436074](https://zhuyaguang-1308110266.cos.ap-shanghai.myqcloud.com/img/image-20240329154436074.png)
+
+#### 2. Modify dnsPolicy
+
+After the deployment of edgemesh is complete, the two environment variables in node-exporter on the edge node are still empty, and it is not possible to access kubernetes.default.svc.cluster.local:443. The reason is that the DNS server configuration in the pod is incorrect. It should be 169.254.96.16, but it is the same as the host's DNS configuration.
+
+```shell
+kubectl exec -it node-exporter-hcmfg -n kubesphere-monitoring-system -- sh
+Defaulted container "node-exporter" out of: node-exporter, kube-rbac-proxy
+$ cat /etc/resolv.conf 
+nameserver 127.0.0.53
+```
+
+Change the dnsPolicy to ClusterFirstWithHostNet, then restart node-exporter.
+
+`kubectl edit ds node-exporter -n monitoring`
+
+      dnsPolicy: ClusterFirstWithHostNet
+      hostNetwork: true
+
+#### 3. Add environment variables
+
+vim /etc/systemd/system/edgecore.service
+
+![image-20240329155133337](https://zhuyaguang-1308110266.cos.ap-shanghai.myqcloud.com/img/image-20240329155133337.png)
+
+```
+Environment=METASERVER_DUMMY_IP=kubernetes.default.svc.cluster.local
+Environment=METASERVER_DUMMY_PORT=443
+```
+
+After modification, restart edgecore
+
+```
+systemctl daemon-reload
+systemctl restart edgecore
+```
+
+**node-exporter is now running!**!!!!
+
+In the edge node, you can find that the data of the edge node has been collected by curling http://127.0.0.1:9100/metrics.
+
+### KubeEdge = 1.17.0 
+
+When deploying version 1.17.0, pay attention that it is necessary to support edge Pods to use InClusterConfig to access Kube-APIServer, so you need to configure the specified cloudCore.featureGates.requireAuthorization=true and cloudCore.modules.dynamicController.enable=true. Details can be found in the [KubeEdge public account article](https://mp.weixin.qq.com/s/Dw2IKRDvOWH52xTOStI7dg)
```

Comment:
English document about inclusterconfig is not available now, I will add it asap.

Reply:
ok